content/40.inequality.md

```diff
@@ -9,6 +9,93 @@ The fact that differences in survival between Black and white patients were no l
+The CFR for adults over 80 has been estimated upwards of 14% or even 20% [@doi:10.1111/jgs.16472].
+Male sex has also been identified as a risk factor for severe COVID-19 outcomes, including death [@doi:10.1136/bmj.m1985; @doi:10.1186/s13293-020-00304-9; @url:https://globalhealth5050.org/the-sex-gender-and-covid-19-project/].
+Early reports from China and Europe indicated that even though the case rates were similar across males and females, males were at elevated risk for hospital admission, ICU admission, and death [@doi:10.1186/s13293-020-00304-9], although data from some US states indicates more cases among females, potentially due to gender representation in care-taking professions [@doi:10.1371/journal.ppat.1008570].
+In older age groups (e.g., age 60 and older), comparable absolute numbers of male and female cases actually suggests a higher rate of occurrence in males, due to increased skew in the sex ratio [@doi:10.1186/s13293-020-00304-9].
+Current estimates based on worldwide data suggest that, compared to females, males may be 30% more likely to be hospitalized, 80% more likely to be admitted to the ICU, and 40% more likely to die as a result of COVID-19 [@url:https://globalhealth5050.org/the-sex-gender-and-covid-19-project/].
+There also may be a compounding effect of advanced age and male sex, with differences time to recovery worst for males over 60 years old relative to female members of their age cohort [@doi:10.1371/journal.ppat.1008520].
+
+Both of these risk factors can be approached through the lens of biology.
+The biological basis for greater susceptibility with age is likely linked to the prevalence of extenuating health conditions such as heart failure or diabetes [@doi:10.1111/jgs.16472].
+Several hypotheses have been proposed to account for differences in severity between males and females.
+For example, some evidence suggests that female sex hormones may be protective [@doi:10.1186/s13293-020-00304-9; @doi:10.1371/journal.ppat.1008570].
+ACE2 expression in the kidneys of male mice was observed to be twice as high as that of females, and a regulatory effect of estradiol on ACE2 expression was demonstrated by removing the gonads and then supplementing with estradiol [@doi:10.1186/2042-6410-1-6; @doi:10.1371/journal.ppat.1008570].
+Other work in mice has shown an inverse association between mortality due to SARS-CoV-1 and estradiol, suggesting a protective role for the sex hormone [@doi:10.1371/journal.ppat.1008570].
+Similarly, evidence suggests that similar patterns might be found in other tissues: preliminary analysis suggested that male patients with aortic valve stenosis may show higher levels of ACE2 expression in the myocardium than females, although this pattern was not found in controls, and research has indicated that females may respond to lower doses of heart medications that act on a pathway shared with ACE2 (Renin angiotensin aldosterone system or RAAS) [@doi:10.1186/s13293-020-00304-9].
```

Comment:
```suggestion
Similarly, evidence suggests that similar patterns might be found in other tissues: preliminary analysis suggested that male patients with aortic valve stenosis showed higher levels of ACE2 expression in the myocardium than females, although this pattern was not found in controls, and research indicated that females responded to lower doses of heart medications that act on a pathway shared with ACE2 (Renin angiotensin aldosterone system or RAAS) [@doi:10.1186/s13293-020-00304-9].
```

Reply:
Not sure what the wording meant by "suggested" here. I wonder if it should just say `Similarly, evidence suggested that similar patterns might be found in other tissues: preliminary analysis showed that male patients with aortic valve stenosis had higher levels...`